lib/timeutils/decode.c

```diff
@@ -228,25 +229,25 @@ __determine_recv_timezone_offset(LogStamp *const timestamp, glong const recv_tim
 }
 
 static void
-__fixup_hour_in_struct_tm_within_transition_periods(LogStamp *stamp, struct tm *tm, glong recv_timezone_ofs)
+__fixup_hour_in_struct_tm_within_transition_periods(LogStamp *stamp, WallClockTime *wct, glong recv_timezone_ofs)
 {
   /* save the tm_hour value as received from the client */
```

Comment:
please update the comment with the new class member name

Reply:
I think this has been eliminated in the meantime. I even grepped the source for tm_hour and I found no occurences.